core/spec.md

```diff
@@ -2353,6 +2361,25 @@ Regardless of how the model is retrieved, the overall format is as follows:
 ```
 
 The following describes the attributes of Registry model:
+
+- `modelversion` <span id="model.modelversion"></span>
+  - Type: String.
+  - OPTIONAL.
+  - The version of the local model of a group or resource.
+  - It is common to use a combination of major and minor version numbers.
+  - Examples: `1.2`
```

Comment:
minor things but let's move `1.2` to a new sub-bullet like the next one has it

Reply:
Will do tomorrow

Reply:
I looked below and I think the consistent form is: 

- Example: `1.2`

Will do the same with compatiblewith.